drivers/rtc/rtc_ll_stm32.c

```diff
@@ -187,12 +187,29 @@ static inline ErrorStatus rtc_stm32_init_alarm(RTC_TypeDef *rtc, uint32_t format
 					LL_RTC_AlarmTypeDef *ll_alarm_struct, uint16_t id)
 {
 	ll_alarm_struct->AlarmDateWeekDaySel = RTC_STM32_ALRM_DATEWEEKDAYSEL_DATE;
-	/* RTC write protection is disabled & enabled again inside LL_RTC_ALMx_Init functions */
+	ll_alarm_struct->AlarmDateWeekDay = 0x01;
```

Comment:
These lines are uncessary & potentially incorrect since the `AlarmDateWeekDaySel` & `AlarmDateWeekDay` fields of `LL_RTC_AlarmTypeDef` are initialized inside `rtc_stm32_init_ll_alrm_struct()`

Reply:
removed